src/components/AdminTable.jsx

```diff
@@ -0,0 +1,71 @@
+import { Table } from 'antd';
+import React from 'react';
+
+function AdminTable() {
+  const columns = [
+    {
+      title: 'Full Name',
+      width: 100,
+      dataIndex: 'name',
+      key: 'name',
+    },
+    {
+      title: 'Age',
+      width: 100,
+      dataIndex: 'age',
+      key: 'age',
+    },
+    {
+      title: 'Column 1',
+      dataIndex: 'address',
+      key: '1',
+      width: 150,
+    },
+    {
+      title: 'Column 2',
+      dataIndex: 'address',
+      key: '2',
+      width: 150,
+    },
+    {
+      title: 'Column 3',
+      dataIndex: 'address',
+      key: '3',
+      width: 150,
+    },
+    {
+      title: 'Action',
+      key: 'operation',
+      width: 200,
+      render: () => (
+        <button type="button" c>
+          action
+        </button>
+      ),
+    },
+  ];
+  const data = [];
+
+  for (let i = 0; i < 100; i += 1) {
```

Comment:
Do not do any sort of calculation on re-render, always memoize calculations to save on performance.

Reply:
This is just for style purpose, I will remove it now.